openmetadata-ui/src/main/resources/ui/src/components/common/DomainLabel/DomainLabelNew.tsx

```diff
@@ -201,7 +169,24 @@ export const DomainLabelNew = ({
         </Typography.Text>
       );
     }
-  }, [activeDomain, domainDisplayName, showDomainHeading, textClassName]);
+  }, [
+    activeDomain,
+    domainDisplayName,
+    showDomainHeading,
+    textClassName,
+    showAll,
+    t,
+  ]);
```

Comment:
we don't need this `t` in the dependency!

Reply:
addreesed